app/docker/components/container-restart-policy/container-restart-policy-controller.js

```diff
@@ -0,0 +1,37 @@
+angular
+  .module('portainer.docker')
+  .controller('ContainerRestartPolicyController', [
+    function ContainerRestartPolicyController() {
+      var ctrl = this;
+
+      this.state = {
+        editMode :false,
+        editModel :{}
+      };
+      
+
+      ctrl.toggleEdit = toggleEdit;
+      ctrl.save = save;
+
+      function toggleEdit() {
```

Comment:
why not simply use `this.toggleEdit = function() {
...
}`

Instead of creating a function and assigning it to `ctrl` ?

Reply:
It makes it easier to see the components' API. you just need to look at the first few lines of the controller.

Reply:
Ok, sounds good to me, any refactoring to other components should use this and we'll keep it in mind for future components.

Reply:
+1